cmd/collectors/zapiperf/zapiperf.go

```diff
@@ -579,13 +582,17 @@ func (me *ZapiPerf) PollData() (map[string]*matrix.Matrix, error) {
 			sValues := met.GetValuesFloat64()
 			pass := met.GetPass()
 			for k := range sValues {
-				pass[k] = sValues[k] > 0
+				if me.isCI {
+					pass[k] = sValues[k] > 0
+				} else {
+					pass[k] = sValues[k] >= 0
+				}
```

Comment:
The condition is flipped here, I think you meant for this to be
```go
if me.isCI {
    pass[k] = sValues[k] >= 0
} else {
    pass[k] = sValues[k] > 0
}
```
but it might be better to use
```go
pass[k] = isCI || sValues[k] > 0
```